libs/sql-schema-describer/src/mysql.rs

```diff
@@ -55,6 +82,22 @@ impl SqlSchemaDescriber {
         names
     }
 
+    fn get_size(&self, schema: &str) -> usize {
+        debug!("Getting table names");
+        let sql = "SELECT 
+      SUM(data_length + index_length) as size 
+      FROM information_schema.TABLES 
+      WHERE table_schema = ?";
+        let result = self.conn.query_raw(sql, &[schema.into()]).expect("get db size ");
+        let size: String = result
+            .first()
+            .map(|row| row.get("size").and_then(|x| x.to_string()).expect("get db size"))
+            .unwrap();
+
+        debug!("Found db size: {:?}", size);
+        size.parse().unwrap()
```

Comment:
Maybe I'm missing context, but we could use `x.to_i64()` instead of `x.to_string()` on line 94 and we wouldn't have to unwrap, I think